src/components/SectionClass.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import ClassItem from './ClassItem';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+
+function SectionClass({sectionIndex, sectionclass, sectionType, bootcampDataCopy, setBootcampDataCopy, section, dayIndex}) {
+    const [classButtonsVisible, setClassButtonsVisible] = useState(false);
+
+    // helper function that maps each element in section array to a new course day
+    const shiftSection = (sectionArray, startDay) => {
```

Comment:
I would prefer that these kinds of functions take `bootcampDataCopy` as a param and return it at the end

Reply:
react with hooks uses a reference to the state inside an event callback like `handleClassShift` but in general we should avoid mutating (changing) values by reference - specifically bc I would say that `shiftSection` is a helper function.